macaroon-minting.xml

```diff
@@ -85,12 +85,12 @@
     
     <section title="Requesting a macaroon">
       <t>The client requests that the server creates a macaroon by
-      issues an HTTP POST request to the path <spanx
+      issuing an HTTP POST request to the path <spanx
       style="verb">/.well-known/macaroons</spanx>.  The request MUST
       have a <spanx style="verb">Content-Type</spanx> header with
       value <spanx
-      style="verb">application/json+macaroon-request</spanx>.  The
-      request entity must be a valid JSON object.</t>
+      style="verb">application/json</spanx>.  The request entity 
```

Comment:
if it should be "application/json;macaroon-reguest' im my oppinion
https://www.w3.org/Protocols/rfc2616/rfc2616-sec3.html#sec3.7